### ISSUE TITLE
Request bug?

### ISSUE THREAD

**Issue by ghost** [dpa99c/cordova-plugin-request-location-accuracy-example#1]
Hello,

I just tried you example and there is a flow that is not working as expected on iOS:
1. We open the app with location services turned off.
2. We "Request location" to trigger the native dialog.
3. We press settings button to go to settings screen.
4. We do not enable location services yet, but go back to the app via "the opened app overview" (by double tapping home).
5. It says we requested successful, in the list Location Services keeps DISABLED.
6. We go back to settings screen via the opened app overview and do enable location settings.
7. We go back to the app and see that Location Services is enabled.

But! When we then close the app, disable our Location Services again and reopen the app. When we now press the button "Request location" the popup of "Successfully requested" shows, but there is not the native popup to go to the settings. Seems like the native popup and the switching of the location services have to keep in sync, if user does not enables it directly and returns to the app, the native popup doesn't show anymore (ever?). Not even after restarting the app.

Any insights?

**Comment by dpa99c** [comment]
This is a slightly different manifestation of the [iOS "Cancel" button caveat](https://github.com/dpa99c/cordova-plugin-request-location-accuracy#ios-cancel-button-caveat).

Similar to that scenario, when the user decides to manually disable Location Services after having enabled it via the native Settings dialog that the app invoked, Apple makes the assumption that the user explicitly did not want Location Services enabled. Therefore, the app is no longer able to invoke and display the native dialog. However, there is no callback or error from iOS that informs the app of this outcome, so the plugin is unable to determine if the native dialog was successfully shown or not. It can only report that it successfully requested the dialog be shown. The rest of the logic is in the hands of Apple and iOS.

So in short, rather than a bug which is solvable, this is another caveat to bear in mind if using this plugin, resulting from limitations imposed by hard-coded logic in iOS.

I will add this scenario to the documented caveats, alongside the Cancel button scenario.

**Comment by ghost** [comment]
I was afraid that this would be the answer. Thanks anyways. Will report like this to the team! Great work on this and diagnostic plugin!

**Comment by kagde** [comment]
Hello guys i am using cordova-plugin-request-location-accuracy plugin but its not working on lollipop android device its working fine latest version like marshmallow   is any one have solution?
